### PR TITLE
Update aspnet-core-integration.md

### PR DESCRIPTION
**Pull request by mrnavrc**
When I tried to implement Quartz in my web application, I missed a short practical example of implementing the package within ASP.NET Core 6 directly in the official manual, so I had to look elsewhere. I think this will help anyone who comes across this package for the first time and doesn't know what to expect.

### CODE REVIEW

File: docs/documentation/quartz-3.x/packages/aspnet-core-integration.md [hunk 1]
```diff
@@ -37,4 +37,47 @@ public void ConfigureServices(IServiceCollection services)
...
+    q.AddTrigger(opts => opts
+        .ForJob(jobKey)
+        .WithIdentity("SendEmailJob-trigger")
+        .WithCronSchedule("0 * * ? * *")); //This Cron interval can be described as "run every minute"
+});
+builder.Services.AddQuartzHostedService(q => q.WaitForJobsToComplete = true);
+```
+For more information on CronTriggers, you can use the tutorial directly from Quartz - [CronTriggers](https://www.quartz-scheduler.net/documentation/quartz-3.x/tutorial/crontriggers.html).
+
+And for easy generation of Cron intervals using UI you can use - [Free Cron Expression Generator & Explainer](https://www.freeformatter.com/cron-expression-generator-quartz.html).
```
Comment (drieseng) [inline docs/documentation/quartz-3.x/packages/aspnet-core-integration.md:83]: I propose to move this to the linked **contriggers** page, if that's ok for @lahma.

Comment (lahma) [reply docs/documentation/quartz-3.x/packages/aspnet-core-integration.md:83]: I think both might be ideal, repetition and maintenance, but people usually miss these. I'd also add http://www.cronmaker.com/ to the mix, maybe link text should contain service name too (freeformatter.com)?

**Comment by lahma** [comment]
Thanks for the PR, documentation updates and improvements are really appreciated!

**Comment by MrDave1999** [comment]
Today I just discovered the existence of this package and I also got lost when integrating Quartz into ASP.NET Core. I hope they will merge this PR soon.

**Pull request merged**